src/builtins.rs

```diff
@@ -120,3 +125,34 @@ pub fn llm(args: Vec<String>, stdin: Option<ChildStdout>) -> Result<String, Box<
 
     Ok("".to_string())
 }
+
+pub fn alias(
+    args: Vec<String>,
+    aliases: &mut HashMap<String, String>,
+) -> Result<String, Box<dyn Error>> {
+    if args.is_empty() {
+        // Print all aliases
+        for (alias, command) in aliases {
+            println!("{}='{}'", alias, command);
+        }
+        Ok("".to_string())
+    } else if args.len() == 1 {
+        let parts: Vec<&str> = args[0].splitn(2, '=').collect();
+        if parts.len() == 2 {
+            let alias = parts[0];
+            let command = parts[1];
+            aliases.insert(alias.to_string(), command.to_string());
```

Comment:
The [reference manual](https://www.gnu.org/software/bash/manual/bash.html#Aliases) does not allow special characters in the name. Not sure if we want to enforce that or not :)